packages/cli/test/integration/generators/repository.integration.js

```diff
@@ -289,6 +289,29 @@ describe('lb4 repository', function() {
       );
     });
 
+    it('allows other connectors', async () => {
+      const files = SANDBOX_FILES.filter(
+        e =>
+          e.path !== 'src/datasources' ||
+          e.file.includes('sqlite-3.datasource.'),
+      );
+      const basicPrompt = {
+        dataSourceClass: 'Sqlite_3Datasource',
+      };
+      return expect(
+        testUtils
+          .executeGenerator(generator)
+          .inDir(SANDBOX_PATH, () =>
+            testUtils.givenLBProject(SANDBOX_PATH, {
+              // Only use the sqlite3 datasource
+              additionalFiles: files,
+            }),
+          )
+          .withPrompts(basicPrompt)
+          .withArguments(' --model Defaultmodel'),
+      ).to.not.be.rejectedWith(/No datasources found/);
```

Comment:
I am wondering if the command exits with another error, would the test pass?
If yes I think omit the message here is better than specify it.